src/demos/shell/demo-app-full-window.html

```diff
@@ -48,8 +48,8 @@
     <main>
       <div class="shell-container">
         <calcite-shell>
-          <calcite-shell-panel slot="primary-panel" position="start" detached>
-            <calcite-panel
+          <calcite-shell-panel slot="primary-panel" position="start" >
+            <!-- <calcite-panel
```

Comment:
Please remove commented out code if no longer applicable. This applies to other lines as well.

Reply:
👍